src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ClientFileMetadataRouteSpec.scala

```diff
@@ -152,16 +153,21 @@ class ClientFileMetadataRouteSpec extends AnyFlatSpec with Matchers with TestReq
     ps.executeUpdate()
   }
 
-  private def checkClientFileMetadataExists(clientFileMetadataId: UUID): Unit = {
-    val sql = "select * from ClientFileMetadata where ClientFileMetadataId = ?;"
+  private def checkClientFileMetadataExists(fileId: UUID): Unit = {
+    val sql = "select * from FileMetadata where FileId = ? AND PropertyName IN (?,?,?,?);"
```

Comment:
Upper case commands "SELECT", "FROM" and "WHERE" for consistency?

Reply:
I've changed these.